syntax = "proto3";

package mesop.components.text_input;

message TextInputType {
    string label = 1;
    string on_change_handler_id = 2;
    string default_value = 3;
}
